src/Thinktecture.Relay.Server/Services/ServerStatisticsWriter.cs

```diff
@@ -60,7 +66,25 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 	/// <inheritdoc/>
 	public override async Task StopAsync(CancellationToken cancellationToken)
 	{
-		await _statisticsWriter.SetShutdownTimeAsync(_serverContext.OriginId, cancellationToken);
+		await _originWriter.SetShutdownTimeAsync(_serverContext.OriginId, cancellationToken);
 		await base.StopAsync(cancellationToken);
 	}
+
+	private async Task ExecuteOriginUpdatesAsync(CancellationToken stoppingToken)
+	{
+		while (!stoppingToken.IsCancellationRequested)
+		{
+			await _originWriter.UpdateLastSeenTimeAsync(_serverContext.OriginId, stoppingToken);
+			await Task.Delay(_statisticsOptions.LastActivityUpdateInterval, stoppingToken);
```

Comment:
The naming of `LastActivityUpdateInterval` should be changed to `OriginLastSeenUpdateInterval`.

Reply:
It is still named `LastActivityUpdateInterval`?